### PR TITLE
[DE] Update `osu!_File_Formats`

### PR DESCRIPTION
**Pull request by The-Last-Cookie**
Updates the article to the latest version.

## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

File: wiki/osu!_File_Formats/de.md [hunk 1]
```diff
@@ -1,34 +1,63 @@
...
+- osu! (und ansonsten nichts, da das Spiel selbst .osz-/.osk-Dateien erzeugen kann)
+
+**Prozess**
 
-.osz/.osk Datei ist ein Dateierweiterung, die extra für osu!.exe zugeschnitten ist. Wenn eine .osz Datei geöffnet wird, wird die Datei von der osu!.exe extrahiert und zum "Songs" Ordner transferiert. Die .osk Datei wird hingegen zum "Skins" Ordner transferiert. Standardmäßig werden Beatmapdateien direkt von der offiziellen Webseite im .osz Dateiformat heruntergeladen. Wie man .osz/.osk Dateien erstellt, zählt als fundamentales Wissen eines Beatmappers/Skinners.
+1. Wie oben, führe deine Dateien in einem Ordner zusammen und benenne diesen.
+   - Wenn du die Beatmap vorher editiert hast, wird normalerweise alles bereits fertig zum Konvertieren nach .osz sein. Für Skins musst du die Dateien vorher kompillieren.
+2. Öffne osu!.
+3. Wenn du eine .osz machen möchtest, dann:
+   - Gehe zu "Zum Editieren" und wähle die Beatmap, die du zu .osz konvertieren möchtest.
```
Comment (respektive) [inline wiki/osu!_File_Formats/de.md:53]:
```suggestion
   - Gehe zu "Edit" und wähle die Beatmap, die du zu .osz konvertieren möchtest.
```
the button in the main menu is just called `Edit` even in german.

**Review by respektive — CHANGES_REQUESTED** [review]
looks good otherwise

File: wiki/osu!_File_Formats/de.md [hunk 2]
```diff
@@ -1,34 +1,63 @@
...
+
+**Benötigte Software:**
+
+- osu! (und ansonsten nichts, da das Spiel selbst .osz-/.osk-Dateien erzeugen kann)
+
+**Prozess**
 
-.osz/.osk Datei ist ein Dateierweiterung, die extra für osu!.exe zugeschnitten ist. Wenn eine .osz Datei geöffnet wird, wird die Datei von der osu!.exe extrahiert und zum "Songs" Ordner transferiert. Die .osk Datei wird hingegen zum "Skins" Ordner transferiert. Standardmäßig werden Beatmapdateien direkt von der offiziellen Webseite im .osz Dateiformat heruntergeladen. Wie man .osz/.osk Dateien erstellt, zählt als fundamentales Wissen eines Beatmappers/Skinners.
+1. Wie oben, führe deine Dateien in einem Ordner zusammen und benenne diesen.
+   - Wenn du die Beatmap vorher editiert hast, wird normalerweise alles bereits fertig zum Konvertieren nach .osz sein. Für Skins musst du die Dateien vorher kompillieren.
```
Comment (Phippe) [inline wiki/osu!_File_Formats/de.md:50]:
```suggestion
   - Wenn du die Beatmap vorher editiert hast, wird normalerweise alles bereits fertig zum Konvertieren nach .osz sein. Für Skins musst du die Dateien vorher kompilieren.
```

File: wiki/osu!_File_Formats/de.md [hunk 3]
```diff
@@ -1,34 +1,63 @@
...
+2. Öffne osu!.
+3. Wenn du eine .osz machen möchtest, dann:
+   - Gehe zu "Edit" und wähle die Beatmap, die du zu .osz konvertieren möchtest.
+   - Öffne "Datei", dann "Paket exportieren ..."
+   - Die .osz wird erzeugt und im Ordner "Exports" deines osu!-Ordners platziert.
+4. Wenn du eine .osk machen möchtest, dann:
+   - Versichere dich als erstes, dass dein Skin alles hat, was dieser haben soll. Du kannst das prüfen, indem du "Skinordner öffnen" wählst, sofern du den Skin zum Konvertieren nutzt.
+   - Dann, im "Optionen"-Menü, klicke auf "Skin auswählen".
+   - Wähle den Skin, den du möchtest im Skinselektionsmenü und klicke auf "Als .osk exportieren".
```
Comment (Phippe) [inline wiki/osu!_File_Formats/de.md:59]:
```suggestion
   - Wähle den Skin, den du möchtest, im Skinselektionsmenü und klicke auf "Als .osk exportieren".
```

**Review by Phippe — CHANGES_REQUESTED** [review]
Looks good to me otherwise 👍

File: wiki/osu!_File_Formats/de.md [hunk 4]
```diff
@@ -1,34 +1,63 @@
...
+| [.osr][Osr Link] | `x-osu-replay` | osu!-[Replay](/wiki/Replay) |
 
-| Dateiformat | Verwendung |
-| :-- | :-- |
-| [.osz][Osz Link] | Ausführbare Beatmapdatei. |
-| [.osk][Osk Link] | Ausführbare Skindatei. |
+## .osz/.osk Datei erstellen
 
-### Beatmap Editor
+.osz/.osk Datei ist ein Dateierweiterung, die extra für osu!.exe zugeschnitten ist. Wenn eine .osz Datei geöffnet wird, wird die Datei von der osu!.exe extrahiert und zum "Songs" Ordner transferiert. Die .osk Datei wird hingegen zum "Skins" Ordner transferiert. Standardmäßig werden Beatmapdateien direkt von der offiziellen Webseite im .osz Dateiformat heruntergeladen. Wie man .osz/.osk Dateien erstellt, zählt als fundamentales Wissen eines Mappers/Skinners.
```
Comment (respektive) [inline wiki/osu!_File_Formats/de.md:15]:
```suggestion
.osz/.osk Datei ist eine Dateierweiterung, die extra für osu!.exe zugeschnitten ist. Wenn eine .osz Datei geöffnet wird, wird die Datei von der osu!.exe extrahiert und zum "Songs" Ordner transferiert. Die .osk Datei wird hingegen zum "Skins" Ordner transferiert. Standardmäßig werden Beatmapdateien direkt von der offiziellen Webseite im .osz Dateiformat heruntergeladen. Wie man .osz/.osk Dateien erstellt, zählt als fundamentales Wissen eines Mappers/Skinners.
```

**Review by respektive — APPROVED** [review]
LGTM

**Review by TicClick — APPROVED** [review]

**Pull request merged**